Makefile

```diff
@@ -640,6 +641,9 @@ HTML_TEMPLATE_FILES = \
 # Don't include anything here that includes llvm headers.
 # Also *don't* include anything that's only used internally (eg SpirvIR.h).
 # Keep this list sorted in alphabetical order.
+#
+# Note that CompilerGlobals.h is deliberately omitted, as we don't
```

Comment:
This comment seems redundant with the one above. There are lots of .h files not in this list.